pelican/__init__.py

```diff
@@ -366,13 +380,13 @@ def parse_arguments(argv=None):
 
     parser.add_argument('-e', '--extra-settings', dest='overrides',
                         help='Specify one or more SETTING=VALUE pairs to '
-                             'override settings. If VALUE contains spaces, '
-                             'add quotes: SETTING="VALUE". Values other than '
-                             'integers and strings can be specified via JSON '
-                             'notation. (e.g., SETTING=none)',
+                             'override settings. VALUE must be in JSON notation: '
+                             'specify string values as SETTING=\'"some string"\'; '
+                             'booleans as SETTING=true or SETTING=false; '
+                             'None as SETTING=null.',
                         nargs='*',
-                        action=ParseDict
-                        )
+                        action=ParseOverrides,
+                        default={})
```

Comment:
It doesn't matter much but technically the default should be an empty list (`[]`) because parser expects a list.

Reply:
`add_argument`'s `default` is "The value produced if the argument is absent from the command line" (https://docs.python.org/3/library/argparse.html#the-add-argument-method). So it's not an input to `ParseOverrides` but the value of `args.overrides`.  Any empty iterable will work, but since `args.overrides` is a dict, the default should be an empty dict.

Reply:
You're right, not sure why I was thinking it would be passed to `ParseOverrides`. Sorry.